src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/command/flow/ingress/of/IngressFlowLoopInstallMultiTableFlowModFactory.java

```diff
@@ -15,18 +15,41 @@
 
 package org.openkilda.floodlight.command.flow.ingress.of;
 
-import org.openkilda.floodlight.command.flow.ingress.IngressFlowSegmentCommand;
+import org.openkilda.floodlight.command.flow.ingress.IngressFlowSegmentBase;
 import org.openkilda.floodlight.switchmanager.SwitchManager;
 import org.openkilda.floodlight.utils.OfFlowModAddMultiTableMessageBuilderFactory;
 import org.openkilda.model.SwitchFeature;
 
 import net.floodlightcontroller.core.IOFSwitch;
+import org.projectfloodlight.openflow.protocol.action.OFAction;
+import org.projectfloodlight.openflow.protocol.instruction.OFInstruction;
 
+import java.util.List;
 import java.util.Set;
 
-public class IngressFlowLoopInstallMultiTableFlowModFactory extends IngressFlowSegmentInstallFlowModFactory {
+public class IngressFlowLoopInstallMultiTableFlowModFactory extends IngressInstallFlowModFactory {
     public IngressFlowLoopInstallMultiTableFlowModFactory(
-            IngressFlowSegmentCommand command, IOFSwitch sw, Set<SwitchFeature> features) {
+            IngressFlowSegmentBase command, IOFSwitch sw, Set<SwitchFeature> features) {
         super(new OfFlowModAddMultiTableMessageBuilderFactory(SwitchManager.FLOW_LOOP_PRIORITY), command, sw, features);
     }
+
+    @Override
+    protected List<OFAction> makeServer42IngressFlowTransformActions(List<Integer> vlanStack) {
+        return null;
+    }
+
+    @Override
+    protected List<OFAction> makeTransformActions(List<Integer> vlanStack) {
+        return null;
+    }
+
+    @Override
+    protected List<OFInstruction> makeMetadataInstructions() {
+        return null;
+    }
+
+    @Override
+    protected OFAction makeOutputAction() {
+        return null;
```

Comment:
As I see this method used as
`applyActions.add(makeOutputAction());`

Will we add null to the list?

Reply:
All these methods are used only to generate regular flow rules. This factory is designed to create loop rules and should not be used for regular flow rules. So it's doesn't matter null or empty lists are used here. Maybe throw UnsupportedOperationException is the better way.

Reply:
I do not want to block PR, but I wondering about NPEs. I agree with UnsupportedOperationException idea

Reply:
Fixed.